media-libs/vulkan-layers/vulkan-layers-1.2.189.0.ebuild

```diff
@@ -13,9 +13,9 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_SUBMODULES=()
 	inherit git-r3
 else
-	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/sdk-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
There's a `sdk-1.2.189.2` tag in Vulkan-ValidationLayers. Should we go for that instead of .0?

I'd prefer a version bump to .2 rather than changing a stable ebuild's distfile.

Reply:
I am unsure, the lunarg site lists `1.2.189.0` as the latest version for both linux and osx and `1.2.189.2` for windows only. I presume there are windows only fixes in those newer tags. Regardless I suspect a new sdk release will be out in the near future, I see a `sdk-1.2.198.0` tag on the gfxreconstruct repo already, but the lunarg site has not been updated yet so maybe its not finalized yet.